test/fixtures/simple_zonal/example.tf

```diff
@@ -17,13 +17,5 @@
 module "example" {
   source = "../../../examples/simple_zonal_with_acm"
 
-  project_id          = var.project_ids[1]
-  cluster_name_suffix = "-${random_string.suffix.result}"
-  region              = var.region
-  zones               = slice(var.zones, 0, 1)
-  network             = google_compute_network.main.name
-  subnetwork          = google_compute_subnetwork.main.name
-  ip_range_pods       = google_compute_subnetwork.main.secondary_ip_range[0].range_name
-  ip_range_services   = google_compute_subnetwork.main.secondary_ip_range[1].range_name
-  operator_path       = "/workspace/acm.yaml"
```

Comment:
Unrelated but looks like we could get rid of https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/test/task_helper_functions.sh and build step too.

Reply:
Let's leave that for a follow-up: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/1239